### ISSUE TITLE
train `sciarg_are` with `SciCo-longformer`

### ISSUE THREAD

**Issue by ArneBinder** [ArneBinder/pie-document-level#443]

**Comment by ArneBinder** [comment]
# first try
- at dfki machine
- single run
- requires `torch>2.0.0`
- based on params from https://github.com/ArneBinder/pie-document-level/pull/381#issuecomment-2671774372
  - but learning rate from SciCo paper and task learning rate 10x of that
- wandb run: [tough-frog-62](https://wandb.ai/arne/dataset-sciarg-task-are-training/runs/u10jmyee)
```
python src/train.py \
experiment=sciarg_are \
transformer_model=allenai/longformer-scico \
+taskmodule.add_global_attention_mask_to_input=true \
model=sequence_classification_with_pooler \
+model.classifier_dropout=0.1 \
model.learning_rate=0.00001 \
model.task_learning_rate=0.0001 \
datamodule.batch_size=128 \
datamodule.num_workers=12 \
trainer.max_epochs=25 \
trainer.min_epochs=25 \
test=true \
trainer=gpu \
datamodule.batch_size=8
```
- status: FINISHED
- result location: `logs/training/runs/dataset-sciarg/task-are/2025-03-19_00-35-38/job_return_value.json`
